test_unstructured/test_setup_utils.py

```diff
@@ -0,0 +1,9 @@
+from setup_utils import load_requirements
+
+
+def test_load_requirements():
+    file = "./files/example.in"
```

Comment:
```suggestion
    file = f"{os.path.dirname(__file__)}/files/example.in"
```
When dealing with tests, the working directory gets wonky. In many situations, it's the project base directory. So this test fails when, for example, I run `pytest test_unstructured/test_setup.py`. Maybe it passes when I `make test`... but I don't always want to run the whole suite. If you follow this suggestion you'd also need to `import os`.